projectforge-wicket/src/main/java/org/projectforge/web/vacation/helper/VacationViewHelper.java

```diff
@@ -238,4 +274,51 @@ private boolean appendFieldset(GridBuilder gridBuilder, final String label, fina
     return true;
   }
 
+  private String getVacationCount(int fromYear, int fromMonth, int toYear, int toMonth, PFUserDO user)
```

Comment:
Auslagern in EmployeeService, damit man es auch an anderen Stellen verwenden kann.